packages/eslint-plugin-office-addins/src/rules/test-for-null-using-isNullObject.ts

```diff
@@ -0,0 +1,155 @@
+import { TSESTree, AST_NODE_TYPES } from "@typescript-eslint/typescript-estree";
+import {
+  Reference,
+  Variable,
+} from "@typescript-eslint/experimental-utils/dist/ts-eslint-scope";
+import { RuleFix, RuleFixer } from "@typescript-eslint/experimental-utils/dist/ts-eslint";
+
+export = {
+  name: "test-for-null-using-isNullObject",
+  meta: {
+    type: <"problem" | "suggestion" | "layout">"problem",
+    messages: {
+      useIsNullObject: "Test {{name}} for null using isNullObject.",
+    },
+    docs: {
+      description: "",
+      category: <
+        "Best Practices" | "Stylistic Issues" | "Variables" | "Possible Errors"
+      >"Possible Errors",
+      recommended: <false | "error" | "warn">false,
+      url:
+        "https://docs.microsoft.com/en-us/office/dev/add-ins/develop/application-specific-api-model?view=powerpoint-js-1.1#ornullobject-methods-and-properties",
+    },
+    schema: [],
+    fixable: <"code" | "whitespace"> "code",
```

Comment:
This too could follow the same category formatting pattern?

Reply:
Handled by prettier.